src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -227,15 +232,11 @@ public static EnergyBalanceInfo ComputeEnergyBalance(IReadOnlyList<OrganelleTemp
     ///   If true, then the required input compounds to run at the given energy balance are stored per energy producer
     /// </param>
     /// <param name="cache">Auto-Evo Cache for speeding up the function</param>
-    public static EnergyBalanceInfo ComputeEnergyBalance(IReadOnlyList<OrganelleTemplate> organelles,
+    public static void ComputeEnergyBalance(EnergyBalanceInfoSimple result, IReadOnlyList<OrganelleTemplate> organelles,
```

Comment:
I think this method needs to be split into two even if that results in code duplication as I think it will be much cleaner to remove the dummy methods from the lite version of energy balance that do absolutely nothing, like `SetupTrackingForRequiredCompounds`. That's not optimal but I think it might be warranted in this very core piece of code that needs maximum performance. We just need to add big enough warning comments about the duplicated code that needs to all be changed if one copy is changed.